src/components/encounter/encounter-form.component.tsx

```diff
@@ -439,6 +442,7 @@ const EncounterForm: React.FC<EncounterFormProps> = ({
             errorFields.push(field);
             field.meta.submission = { ...(field.meta.submission || {}), errors };
             formHasErrors = true;
+            setFieldErrors((prevErrors: ValidationResult[]) => [...prevErrors, ...errors]);
```

Comment:
How would this value ever get cleared?

Reply:
@ibacher  Would [this](https://github.com/openmrs/openmrs-form-engine-lib/pull/245/files#diff-6cea27fb5ce9227dfd782d3b87b9cca868f1ce1fa963bcb1a7b329fe1f02aacfR446) suffice to remove them everytime we are validating/saving the form!

Reply:
Yes, that seems to work!